deploy/job-manager/nginx.conf.ctmpl

```diff
@@ -41,7 +46,7 @@ http {
 
```

Comment:
To avoid the nginx problems 111: https://serverfault.com/a/317476, we'd better also change 
```
location /api {
             proxy_pass http://localhost:8190;
         }
``` to 
```
location /api {
             proxy_pass http://127.0.0.1:8190;
         }
```

Reply:
I got this error `connect() failed (111: Connection refused) while connecting to upstream, client: 10.4.4.1, server: , request: "GET /api/v1/capabilities HTTP/1.1", upstream: "http://[::1]:8190/api/v1/capabilities"` during the testing, and it turned out that`[::1] is localhost IPv6 address`